app/config/urls/root.py

```diff
@@ -45,6 +46,7 @@ def handler500(request):
 
 urlpatterns = [
     path("", HomeTemplate.as_view(), name="home"),
+    path("github-install-complete", post_install_redirect),
```

Comment:
Missing a trailing slash and would be nicer to include the urls from the app rather than importing a view from another app here.